src/routes/auth.js

```diff
@@ -1,10 +1,39 @@
-const M = require('../middle-end');
-const { default: connectedAuthWrapper } = require('redux-auth-wrapper/connectedAuthWrapper');
-const NotFoundPage = require('../components/NotFoundPage');
-
-exports.authenticate = connectedAuthWrapper({
-    wrapperDisplayName: 'UserIsAuthenticated',
-    authenticatedSelector: (state) => M.selectors.auth.getIsAuthenticated(state),
-    authenticatingSelector: (state) => !M.selectors.auth.getHasAuthenticationSettled(state),
-    FailureComponent: NotFoundPage
+// const M = require('../middle-end');
+// const { default: connectedAuthWrapper } = require('redux-auth-wrapper/connectedAuthWrapper');
+// const NotFoundPage = require('../components/NotFoundPage');
+
+// exports.authenticate = connectedAuthWrapper({
+//     wrapperDisplayName: 'UserIsAuthenticated',
+//     authenticatedSelector: (state) => M.selectors.auth.getIsAuthenticated(state),
+//     authenticatingSelector: (state) => !M.selectors.auth.getHasAuthenticationSettled(state),
+//     FailureComponent: NotFoundPage
+// });
+
+const { connectedRouterRedirect } = require('redux-auth-wrapper/history4/redirect');
```

Comment:
I'm a little curious where the `connectedAuthWrapper` was falling down (versus the `history4/redirect`).  Since routing is tied to redux, the `connectedAuthWrapper` might be a little more fitting if it's straightforward to make it work.

Reply:
I did some research on the differences between `connectedAuthWrapper` and `connectedRouterRedirect` and from my understanding, `connectedAuthWrapper`'s main purpose is for hiding or rendering different components based on authentication status while `connectedRouterRedirect` is mainly used for redirection which I _think_ is what we're trying to go for here with how we're using `Auth.authenticate()` HOC.

There isn't much description as to what each of these methods do and my research was mostly based on how they were used in the APIs getting started [section](https://mjrussell.github.io/redux-auth-wrapper/docs/Getting-Started/ReactRouter4.html)